src/transform/FooterContainer.css

```diff
@@ -28,6 +28,7 @@
   display: table;
   width: 100%;
   border-collapse: collapse;
+  line-height:22px;
```

Comment:
This is a more general comment. Sizes like these are probably better specified in em (or even better in rem).